modules/jankyborders/darwin.nix

```
@@ -0,0 +1,14 @@
+{ mkTarget, config, ... }:
+mkTarget {
+  name = "jankyborders";
+  humanName = "JankyBorders";
+
+  configElements =
+    { colors }:
+    {
+      services.jankyborders = {
+        active_color = config.lib.stylix.mkOpacityHexColor colors.base0D config.stylix.opacity.desktop;
+        inactive_color = config.lib.stylix.mkOpacityHexColor colors.base03 config.stylix.opacity.desktop;
+      };
+    };
```

The `config.stylix.opacity` attribute set should be guarded by requesting it:

```suggestion
    { colors, opacity }:
    {
      services.jankyborders = {
        active_color = config.lib.stylix.mkOpacityHexColor colors.base0D opacity.desktop;
        inactive_color = config.lib.stylix.mkOpacityHexColor colors.base03 opacity.desktop;
      };
    };
```

This could arguably be further improved to:

```suggestion
    { colors, opacity }:
    {
      services.jankyborders = let
        mkOpacityHexColor =
          lib.flip config.lib.stylix.mkOpacityHexColor opacity.desktop;
      in {
        active_color = mkOpacityHexColor colors.base0D;
        inactive_color = mkOpacityHexColor colors.base03;
      };
    };
```